designs/virtualization/4.7/add-os-image-updates/add-os-image-to-common-template 4.7.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: Virtualization
+version: 4.7
+---
+# Add an operating system image to a common template
+
+## About
+
+This is an update for the 4.6 version.
+This will cover an update for:
+- the "Operating system" dropdown
+- "Source" dropdown name, placement and options
+
+## OS dropdown
+
+![os dropdown](img/Admin-create-4.7-0-1.jpg)
+The dropdown options will be ordered by two sections, promoting the options with an available image above the ones without.
+This dropdown will also have a search field to avoid looking for a specific OS in two different lists.
+
+## "source" dropdown
+
+![source dropdown](img/Admin-create-4.7-0-2.jpg)
+The "source" field will be renamed to "Boot from".
+It will be indent to the Operating System dropdown.
+
+![source dropdown](img/Admin-create-4.7-0-3.jpg)
```

Comment:
The green text here can probably just go in the description below the title 
The PVCs can also be marked as available across the cluster for this operating selection.